### PR TITLE
docs - add info about querying the pg_stat_last_operation tbl

### PR DESCRIPTION
**Pull request by lisakowen**
enhance the docs for pg_stat_last_operation table to describe the preferred mechanism for querying (objid, not stasubtype); also added an example with sample output.

added this info to the reference page, below the table of fields, as this table isn't referenced elsewhere in the docs.

### CODE REVIEW

File: gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml [hunk 1]
```diff
@@ -7,4 +7,34 @@ about database objects (tables, views, etc.).</p><table id="hr138428"><title>pg_
...
+              detailed information about the operation in the logs.</entry></row></tbody></tgroup></table>
+
+    <p>The <codeph>pg_stat_last_operation</codeph> table contains metadata tracking
+      information about operations on database objects. This information includes the
+      object id, DDL action, user, type of object, and operation timestamp. Greenplum
+      Database updates this table when a database object is created, altered,
+      truncated, vacuumed, analyzed, or partitioned, and when privileges are granted
+      to an object.</p>
+    <p>If you want to track the operations performed on a specific object, use the
+      <codeph>objid</codeph>. Because the <codeph>stasubtype</codeph> can identify
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml:19]: objid -> objid value
stasubtype -> stasubtype value

File: gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml [hunk 2]
```diff
@@ -7,4 +7,34 @@ about database objects (tables, views, etc.).</p><table id="hr138428"><title>pg_
...
+      either the type of object operated on or the subclass of operation performed,
+      it is not a suitable parameter when querying the
+      <codeph>pg_stat_last_operation</codeph> table.</p>
+    <p>The following example creates and replaces a view, and then shows how to use
+      the <codeph>objid</codeph> as a query parameter on the
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml:24]: same edit here.  Either add "value" or just remove "the"

File: gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml [hunk 3]
```diff
@@ -7,4 +7,34 @@ about database objects (tables, views, etc.).</p><table id="hr138428"><title>pg_
...
+testdb=# CREATE OR REPLACE VIEW trial AS SELECT * FROM gp_segment_configuration;
+CREATE VIEW
+testdb=# SELECT * FROM pg_stat_last_operation WHERE objid='trial'::regclass::oid;
+ classid | objid | staactionname | stasysid | stausename | stasubtype |            statime            
+---------+-------+---------------+----------+------------+------------+-------------------------------
+  1259  | 24735 | CREATE         |       10 | gpadmin    | VIEW       | 2020-04-07 16:44:28.808811+00
+  1259  | 24735 | ALTER          |       10 | gpadmin    | SET        | 2020-04-07 16:44:38.110615+00
+(2 rows)</codeblock>
+    <p>Notice that the <codeph>pg_stat_last_operation</codeph> table entry for the         view <codeph>REPLACE</codeph> operation specifies the <codeph>ALTER</codeph>
+      <codeph>staactionname</codeph> and the <codeph>stasubtype</codeph> named
```
Comment (dyozie) [inline gpdb-doc/dita/ref_guide/system_catalogs/pg_stat_last_operation.xml:37]: Edit this to something like:

ALTER action (staactionname) and the SET subtype (stasubstype).

**Comment by gaos1** [comment]
Thanks. LGTM.

**Pull request merged**